ONSdigital/eq-performance-test
# Switch to ruby-jmeter and update to work with latest Star Wars survey

### What is the context of this PR?

Switched this to ruby-jmeter (which we used in the Alpha) to generate JMeter XML files and updated it to work against the current Star Wars survey in pre-prod. Default setup is to test 300 users with 10 second +/- 5 second wait times.

This also switches to using the /dev page to generate the token and introduces a step of producing errors on purpose before fixing them and onto final submission.
### How to review
- Checkout and follow instructions in README.md
- Run against pre-prod and verify tests pass (may not be 100% success yet)
- See if you can make sense of the Ruby code.
### Who worked on the PR

@collisdigital
